CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### App Center
 
 * **[Fix]** Fix an `IncorrectContextUseViolation` warning when calculating screen size on Android 11.
+* **[Feature]** `SQLiteUtils` now executes sql through `SQLiteStatement`.
```

Comment:
Please explain how exactly this affects to the SDK's behaviour. For now it's not clear and I do not see why do we need this change